frame/fast-unstake/src/migrations.rs

```diff
@@ -0,0 +1,74 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+pub mod v1 {
+	use crate::{types::BalanceOf, *};
+	use frame_support::{
+		storage::unhashed,
+		traits::{Defensive, Get, GetStorageVersion, OnRuntimeUpgrade},
+		weights::Weight,
+	};
+	use sp_staking::EraIndex;
+
+	pub struct MigrateToV1<T>(sp_std::marker::PhantomData<T>);
```

Comment:
Correct me if I’m wrong, but aren’t we doing the migration via pallet code itself?

Reply:
removed that now, thanks for spotting!